app/namespaces/test_adjustment/test_adjustment_service.py

```diff
@@ -73,19 +74,26 @@ def __init__(self, model_code=testadj_model_code, model_name='testadj_binomial_s
     def get_stan_model_cache(self, model_code: str, model_name: str = 'anon_model', **kwargs: Dict) -> pystan.StanModel:
         """Use just as you would `pystan.StanModel`"""
 
-        # Create filepath of cached model
+        # Get working directory and extract system path of iit-backend from it
+        abs_filepath_curr_dir = os.getcwd()
+        proj_root_abs_path = abs_filepath_curr_dir.split("iit-backend")[0]
+
+        # Create model absolute path
         code_hash = md5(model_code.encode('ascii')).hexdigest()
-        cache_fn = f'app/namespaces/test_adjustment/stanmodelcache-{model_name}-{code_hash}.pkl'
+        cache_fn = f'{proj_root_abs_path}iit-backend/app/namespaces/test_adjustment/stanmodelcache-{model_name}-{code_hash}.pkl'
+
+        # Get relative path of model cache (necessary because funtion will be called from different files)
+        rel_path = os.path.relpath(cache_fn, abs_filepath_curr_dir)
```

Comment:
I don't get why having the relative path is advantageous over having the absolute path if we're trying to refer to this cache from different files. In fact, isn't having the relative path more confusing and hard to work with?

Reply:
changed to just use abs path.